Add docs for disabling geoIP (#5765)
PostHog/posthog.com: contents/docs/libraries/python/index.mdx

# returns string or None
```

**Default overridden properties**

> Before posthog-python v3.0, by default we added geoip properties to all incoming events, and for feature flag evaluation, based on the IP address of the request. This isn't ideal since it uses your server IP address, rather than the user's, leading to incorrect location resolution.

Reviewer: ```suggestion
Before posthog-python v3.0, we added GeoIP properties to all incoming events by default. We also added them for feature flag evaluation, based on the IP address of the request. This isn't ideal since they are created based on your server IP address, rather than the user's, leading to incorrect location resolution.
```

> As of posthog-python v3.0, the default now is to disregard the server IP.

Reviewer: ```suggestion
As of posthog-python v3.0, the default now is to disregard the server IP, not add the GeoIP properties, and not use the values for feature flag evaluations.
```

> You can go back to previous behaviour by doing so:

Reviewer: ```suggestion
You can go back to previous behaviour by doing setting the `disable_geoip` argument in your initialization to `False`:
```
